### PR TITLE
Tweaking preset positions

### PR DESCRIPTION
**Pull request by UditS08**
1. Reduced claw min to 8, to make claw grip stronger
2. Adjusted slide extend, slide rotate and claw rotate value for the presets
3. Removed (commented out) the safe drive when slides are rotated down

### CODE REVIEW

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/drive/ClawSlide.java [hunk 1]
```diff
@@ -41,26 +41,26 @@ public ClawSlide(
...
                 rightSlide);
         this.claw = new Claw(
-             clawRotLeft,
-             clawRotRight,
-             clawArmLeft,
-             clawArmRight);
+                clawRotLeft,
+                clawRotRight,
+                clawArmLeft,
+                clawArmRight);
```
Comment (zyxkad) [inline TeamCode/src/main/java/org/firstinspires/ftc/teamcode/drive/ClawSlide.java]: Can you change the indents to four spaces?
```suggestion
            clawRotLeft,
            clawRotRight,
            clawArmLeft,
            clawArmRight);
```

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/teleop/AMainTeleOp.java [hunk 2]
```diff
@@ -130,11 +130,11 @@ public void loop() {
...
-        boolean safeDrive = this.clawSlide.slideRotate.getLeftPosition() < 850 || this.clawSlide.claw.getLeftRotAngle() < 100;
-        if (safeDrive) {
-            this.drive.shift(this.gamepad1.left_stick_x, this.gamepad1.left_stick_y);
-            this.drive.rotate(this.gamepad1.right_stick_x * 0.6);
-        }
+//        boolean safeDrive = this.clawSlide.slideRotate.getLeftPosition() < 850 || this.clawSlide.claw.getLeftRotAngle() < 100;
+//        if (safeDrive) {
+        this.drive.shift(this.gamepad1.left_stick_x, this.gamepad1.left_stick_y);
+        this.drive.rotate(this.gamepad1.right_stick_x * 0.6);
+//        }
```
Comment (zyxkad) [inline TeamCode/src/main/java/org/firstinspires/ftc/teamcode/teleop/AMainTeleOp.java]:
```suggestion
        boolean safeDrive = true || this.clawSlide.slideRotate.getLeftPosition() < 850 || this.clawSlide.claw.getLeftRotAngle() < 100;
        if (safeDrive) {
            this.drive.shift(this.gamepad1.left_stick_x, this.gamepad1.left_stick_y);
            this.drive.rotate(this.gamepad1.right_stick_x * 0.6);
        }
```

**Review by zyxkad — CHANGES_REQUESTED** [review]

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/teleop/AMainTeleOp.java (hunk 2 above)
Comment (UditS08) [reply TeamCode/src/main/java/org/firstinspires/ftc/teamcode/teleop/AMainTeleOp.java]: I was thinking that we will need to be able to drive while the claw is down, this is why I thought of turning safe drive off.

Comment (zyxkad) [reply TeamCode/src/main/java/org/firstinspires/ftc/teamcode/teleop/AMainTeleOp.java]: yeah but make the condition always true is better than commend the if out, in styles

Comment (UditS08) [reply TeamCode/src/main/java/org/firstinspires/ftc/teamcode/teleop/AMainTeleOp.java]: What if I delete the commented code?

Comment (zyxkad) [reply TeamCode/src/main/java/org/firstinspires/ftc/teamcode/teleop/AMainTeleOp.java]: hm okay

**Review by zyxkad — APPROVED** [review]

**Pull request merged**